src/cmd/update.rs

```diff
@@ -0,0 +1,94 @@
+use std::env;
+use std::fs;
+use std::fs::Permissions;
+use std::os::unix::prelude::*;
+use std::path::Path;
+
+use crate::tempfile::Tempfile;
+use anyhow::{anyhow, Context, Result};
+use os_info::{Info, Type};
+use serde::Deserialize;
+
+const VERSION: &str = env!("CARGO_PKG_VERSION");
+const LATEST_RELEASE_URL: &str = "https://api.github.com/repos/sbstp/kubie/releases/latest";
+const FILENAME: &str = "kubie";
+
+#[derive(Debug, Deserialize)]
+pub struct Release {
+    tag_name: String,
+    prerelease: bool,
+    assets: Vec<Asset>,
+}
+
+#[derive(Debug, Deserialize)]
+struct Asset {
+    name: String,
+    browser_download_url: String,
+    state: String,
+}
+
+pub fn update() -> Result<()> {
+    let latest_release: Release = get_latest_release()?;
+    let latest_version = latest_release.tag_name;
+    if latest_version.eq(&format!("v{}", VERSION)) {
```

Comment:
You shouldn't have to use `.eq`, `==` should work here as the `eq` function is the `==` operator, overloaded.

Reply:
Done